quickstart-mongodbatlas/beta-launch.md

```diff
@@ -0,0 +1,97 @@
+
+![atlas-cfn](https://gist.githubusercontent.com/jasonmimick/68f402c378ed364ea1684fda1a7ef5d2/raw/3094b7a2f77d4a8d3e0a8acf4a876844acb39685/atlas-cfn.png "atlas-cfn") MongoDB Atlas Cloud Formation Custom Resources
+=====
+
+# (Pre)-Beta Launch "README"
+
+This note is intended for early adopters and testers interested in trying out the brand new MongoDB Atlas custom resources for Amazon AWS Cloud Formation. 
+Note: this software and the Cloud Formation custom resource SDK from AWS are both still pre-GA products.
+
+![Launch the MongoDB Atlas AWS Quickstart](https://github.com/jasonmimick/quickstart-mongodbatlas/workflows/Launch%20the%20MongoDB%20Atlas%20AWS%20Quickstart/badge.svg?event=workflow_dispatch)
+
+## Getting Started
+
+1. Clone our CFN resources from the source repository: 
+
+```bash
+git clone https://github.com/mongodb/mongodbatlas-cloudformation-resources
+```
+
+2. Setup the `aws` cli as usual, then you can use the helper script to push the resource into the AWS region of your choice.
+
+```bash
+cd mongodbatlas-cloudformation-resources
+
+./util/atlas-cfn-deploy/atlas-cfn-deploy.py --region=us-east-1 all+
+```
+
+There is a helper tool to scrub all the "mongodb-atlas-*-role-stack" stacks: [/util/atlas-cfn-stack-cleaner.sh](/util/atlas-cfn-stack-cleaner.sh).
+
+3. Create a secret with a MongoDB Atlas API Key for CFN to use:
```

Comment:
Highly recommend linking to the docs for how to create a key.

Reply:
:ballot_box_with_check: provisioned some copy from sir terraform, thanks.